ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml

```
@@ -1,3 +1,8 @@
+base_images:
+  upi-installer:
+    name: "4.12"
```

```suggestion
    name: "scos-4.12"
```
same change is needed on all of the others too